packages/client/src/Client.ts

```diff
@@ -130,6 +130,7 @@ export class Client extends EventEmitter {
       const connectToPeer = (documentId: DocumentId, userName: UserName) => {
         const peer = this.get(userName)
         if (peer.has(documentId)) return // don't add twice
+        peer.set(documentId, { socket: null })
```

Comment:
there was a race condition here where you could spam .join before the first socket opened